src/TrayIcon.cpp

```diff
@@ -114,12 +115,33 @@ TrayIcon::TrayIcon(const QString &filename, QWindow *parent)
 
     menu->addAction(viewAction_);
     menu->addAction(quitAction_);
+
+    QString toolTip = QLatin1String("nheko");
+    QString profile = UserSettings::instance()->profile();
+    if (!profile.isEmpty()) {
+        toolTip.append(tr("\nProfile: %1").arg(profile));
```

Comment:
Alternatively we could do it like in the room title: "nheko | myotherprofile". But I am fine with either solution.